pulseapi/entries/views.py

```diff
@@ -117,6 +117,17 @@ class EntryView(RetrieveAPIView):
     pagination_class = None
 
 
+class FavoriteEntries(ListAPIView):
```

Comment:
I didn't inherit from `EntriesListView` because I didn't want this to be POSTable, but we could use some of the filtering options that view has. Thoughts on this?